src/notifications/NotificationsManager.ts

```diff
@@ -161,26 +353,210 @@ class NotificationsManager {
    */
   @ready(new notificationsErrors.ErrorNotificationsNotRunning())
   public async sendNotification(
-    nodeId: NodeId,
-    data: NotificationData,
-  ): Promise<void> {
-    const notification: Notification = {
+    {
+      nodeId,
+      data,
+      retries = 64,
+    }: {
```

Comment:
Avoid magic numbers, If we need a default here then take it from `this.defaultRetries` and set that in the constructor and `createX`. I'm not sure if we need to go as far as adding the default value for this to the config. so it should be fine to have the magic number specified in the `createX` configuration.

Reply:
this has been done.